pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -154,6 +146,7 @@ stdenv.mkDerivation rec {
     isocodes
     json-glib
     libnotify
+    libdbusmenu-gtk3
```

Comment:
The upstream for this dependency is [basically dead](https://launchpad.net/libdbusmenu) and in the past we wanted to [get rid of it](https://github.com/NixOS/nixpkgs/issues/96420) in favour of Debian forks. But it looks like that is now dead as well: https://github.com/AyatanaIndicators/libdbusmenu

Reply:
Is there a course of action you'd suggest I take here? I hate to add new dependencies on dead libraries, but I'd also hate to regress on appindicator support.

Reply:
Looking at https://github.com/ibus/ibus/commit/8923456a5d924c640e70e3755ea60aea84246910, I guess we have to keep it for now.